### DISCUSSION TITLE
Pre-built OS, ISO, virtual machine

### DISCUSSION THREAD

**Discussion by jasonpearce** [runtipi/runtipi-appstore#2734] [Ideas]
Take your project down to the OS level. 

Create a hardened open source Linux distribution that is fully optimized for running as a personal server. Something like StartOS from Start9Labs. You design and package a minimal OS that is best optimized to run Tipi, Docker, and its apps. You provide users an .ISO file that they install on a USB drive for deployment to new or refurbished PCs. 

https://github.com/Start9Labs/start-os is what I'm suggesting for you to consider, but with your Tipi interface and app store. The new machine should work with a monitor or keyboard via a built-in browser, or headless as a server.

Just an idea. And I understand it could be much more difficult to implement than what you currently offer.

**Comment by nicotsx** [comment]
Thanks for the suggestion @jasonpearce, I think this is outside the scope of our project. Creating and maintaining a whole OS to run docker containers doesn't seem like an optimal approach. Tipi can be installed on the OS of your choice with a one-liner command and we don't want to complexify this

**Comment by jasonpearce** [comment]
Thank you for considering this idea. I'm closing to remove it from the list.
